webapp/src/components/systemConsole/questions/questions.tsx

```diff
@@ -22,7 +22,7 @@ const DEFAULT_SURVEY_MESSAGE_TEXT = 'Please take a few moments to help us improv
 
 function SurveyQuestions({id, setSaveNeeded, onChange, config, setInitialSetting}: CustomSettingChildComponentProp) {
     const [questions, setQuestions] = useState<Question[]>([]);
-    const [surveyMessageText, setSurveyMessageText] = useState<string>(DEFAULT_SURVEY_MESSAGE_TEXT);
+    const [surveyMessageText, setSurveyMessageText] = useState<string>();
```

Comment:
I don't get the changes in this file. Why these changes?

Reply:
I'm using `surveyMessageText` as a input `defaultValue`. The way react deals with input placeholder is that it doesn't re-render the inputs on change in `defaultValue`. So, if I initialise the state with `DEFAULT_SURVEY_MESSAGE_TEXT `, then later in a userEffect I change it to a value from the saved config, the input isn't updated as it should based on documentation. So, I removed the default value from state variable and let the `useEffect` set either the default or the saved value from config.

Without this, the saved value from config never shows up.